cg/store/filters/status_case_filters.py

```diff
@@ -200,6 +200,30 @@ def filter_compressible_cases(cases: Query, **kwargs) -> Query:
     return cases.filter(Case.is_compressible)
 
 
+def get_not_received_cases(cases: Query, **kwargs) -> Query:
+    return cases.filter(Sample.received_at != None).distinct()
+
+
+def get_received_cases(cases: Query, **kwargs) -> Query:
+    return cases.filter(Sample.received_at != None).distinct()
```

Comment:
Maybe a bit early but these filters are identical. I would guess `get_received_cases` should have a "=="?